sw/airborne/firmwares/rotorcraft/guidance/guidance_indi_hybrid.c

```diff
@@ -175,6 +182,7 @@ Butterworth2LowPass roll_filt;
 Butterworth2LowPass pitch_filt;
 Butterworth2LowPass thrust_filt;
 Butterworth2LowPass accely_filt;
+Butterworth2LowPass secondary_airspeed_filt;
```

Comment:
I think this "secondary_airspeed" name is misleading. It a regular filter on the airspeed data (that can be filter or not depending on the driver), not a second source of measures. So I recommend to remove the `secondary` from all names.

Reply:
Agreed: this was changed into ```guidance_indi_airspeed_filt```